src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -39,10 +40,15 @@ class Route
     public function __construct(array $data)
     {
         if (isset($data['value'])) {
-            $data['path'] = $data['value'];
+            $data[is_array($data['value']) ? 'locales' : 'path'] = $data['value'];
```

Comment:
does this mean it's possible to add `locales=...` in an annotation and have it work?
is this desired? if yes, it should be covered by a test case (but I'd say it shouldn't so that there is only one way to do it, isn't it?)

Reply:
The `value` is the first value so we map it to the correct property. Because in yaml we now use `path:` as the key I asked @weaverryan what to do about the annotation case, we figured it would be best to keep it inline. So for the annotation you can use `path=LOCALISED_VALUE`.